cmake/RippledCompiler.cmake

```diff
@@ -73,6 +73,7 @@ if (MSVC)
       $<$<CONFIG:Debug>:-MTd>
       $<$<NOT:$<CONFIG:Debug>>:-MT>
       $<$<BOOL:${werr}>:-WX>
+      -DOPENSSL_SUPPRESS_DEPRECATED
```

Comment:
I don't think this shouldn't be here?
Below in `target_compile_definitions()` should suffice.